game-core/src/main/java/games/strategy/engine/auto/update/TutorialMapCheck.java

```diff
@@ -8,8 +8,8 @@
 class TutorialMapCheck {
   static void checkForTutorialMap() {
     final MapDownloadController mapDownloadController = ClientContext.mapDownloadController();
```

Comment:
`mapDownloadController` is now unused (and thus also the `ClientContext` import).

Reply:
Fixed in: 8a8845793